django/db/backends/base/schema.py

```diff
@@ -433,6 +434,21 @@ def add_field(self, model, field):
         db_params = field.db_parameters(connection=self.connection)
         if db_params['check']:
             definition += " " + self.sql_check_constraint % db_params
+        if field.remote_field and self.connection.features.supports_foreign_keys and field.db_constraint:
+            constraint_suffix = "_fk_%(to_table)s_%(to_column)s"
```

Comment:
Please use single quotes.

Reply:
OK. For what it's worth, this file is all over the place in that respect... in fact it seems _most_ of the strings use double quotes (I was doing my best to follow convention, I swear). But I can obviously change this to use single quotes.

Reply:
We prefer single quotes, but again there is no point in doing unrelated refactoring of existing code (see [coding-style](
https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)). Just a nitpick.

Reply:
This is done.